src/main/resources/allowed-labels.properties

```diff
@@ -0,0 +1,71 @@
+# This file lists all the labels that are whitelisted
+
+administrative-monitor
+analysis
+android
+aws
+bitbucket
+bitbucket-server
+builder
+buildwrapper
+ca-apm
+cli
+cloud
+cluster
+cmp
+codebuild
+credential
+database
+deployment
+deprecated
+devops
+dotnet
+emailext
+external
+file
+github
+gitlab
+groovy-related
+ios
+library
+listview-column
+logging
+maven
+misc
+must-be-labeled
+notification
+notifier
+npm
+page-decorator
+parameter
+performance
+pipeline
+pipeline-graph-analysis
+plugin-external
+plugin-misc
+plugin-post-build
+plugin-test
+post-build
+python
+queue
+report
+rocketchatnotifier
+ruby
+runcondition
+scala
+scm
+scm-related
+security
+slaves
```

Comment:
```suggestion
```

I suggest replacing by agents and then doing a synonym support

Reply:
I'm absolutely on board to remove it, but it means anything with the existing label won't show up.

Or should I add a synonym support now for this PR?

Reply:
Can be done in a follow-up